packages/expo-dev-launcher/bundle/screens/ExtensionsScreen.tsx

```diff
@@ -64,20 +82,42 @@ export function ExtensionsScreen({ navigation }: ExtensionsScreenProps) {
                 <View px="small">
                   <Text size="small" align="center">
                     Extensions allow you to customize your development build with additional
-                    capabilities.
+                    capabilities.{' '}
+                    <Text
+                      size="small"
+                      style={{ textDecorationLine: 'underline' }}
+                      onPress={() =>
+                        Linking.openURL(`https://docss.expo.dev/development/extensions/`)
+                      }
+                      accessibilityRole="link">
+                      Learn more.
+                    </Text>
                   </Text>
-                </View>
 
-                <Spacer.Vertical size="small" />
+                  {/* <Spacer.Vertical size="small" />
 
-                <View align="centered">
-                  <Button.ScaleOnPressContainer bg="ghost" rounded="small">
-                    <View border="default" px="small" py="2" rounded="small">
-                      <Button.Text color="ghost" weight="semibold" size="small">
-                        Learn More
-                      </Button.Text>
-                    </View>
-                  </Button.ScaleOnPressContainer>
+                  <View align="centered">
+                    <Button.ScaleOnPressContainer bg="ghost" rounded="small">
+                      <View border="default" px="small" py="2" rounded="small">
+                        <Button.Text color="ghost" weight="semibold" size="small">
+                          Learn More
+                        </Button.Text>
+                      </View>
+                    </Button.ScaleOnPressContainer>
+                  </View> */}
+
+                  <Spacer.Vertical size="medium" />
+
+                  <Text size="small" align="center">
+                    If you would like to extend the display on this screen{' '}
```

Comment:
```suggestion
                    If you would like to extend the display on this screen
```

Reply:
I think this is needed for proper spacing w/ the next nested text element